tests/test_app/fixtures/empty-fixture.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
That's what "empty" looks like in JSON! :smirk: Glad we're using Python. (Just kidding!)

Reply:
Tried a truly empty json file, but Django didn't like it, so we end up with this *empty fixture* :)